bftengine/tests/simpleStorage/FileStorage.cpp

```diff
@@ -0,0 +1,250 @@
+// Concord
+//
+// Copyright (c) 2018 VMware, Inc. All Rights Reserved.
+//
+// This product is licensed to you under the Apache 2.0 license (the "License").
+// You may not use this product except in compliance with the Apache 2.0
+// License.
+//
+// This product may include a number of subcomponents with separate copyright
+// notices and license terms. Your use of these subcomponents is subject to the
+// terms and conditions of the subcomponent's license, as noted in the LICENSE
+// file.
+
+#include "FileStorage.hpp"
+#include "ObjectsMetadataHandler.hpp"
+
+#include <stdio.h>
```

Comment:
This works fine, but why not use the C++ libraries?

Reply:
Actually, I don't need it. Removed.

Reply:
I see, leaving it as is.